src/components/header/Header.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import menu from "./HeaderMenu";
+import logo from "../header/logo.svg";
+import Titles from "./titles";
+
+const Header = () => {
+  const [open, setOpen] = useState(false);
+
+  const hambClick = () => {
+    const displ1 = document.querySelector(".displ1");
```

Comment:
It is bad practice to access elements outside of component. if a component has logic that affect other places, they should be combined